hack/prepare.sh

```diff
@@ -171,33 +206,60 @@ function get_ipv6() {
 	done < <(ip -6 -o addr show lo | awk '{print $4}')
 }
 
+
+function get_pf_mac() {
+	local pci_dev=${devs[$1]}
+	local pf=$(get_ifname $1)
+	cat /sys/bus/pci/devices/$pci_dev/net/$pf/address
+}
+
 function make_config() {
 	if [[ "$IS_X86_WITH_BLUEFIELD" == "true" ]]; then
 		log "Skipping config file creation on AMD/Intel 64-bit host with Bluefield"
 		return
 	fi
-	: > "$CONFIG"
+
 	{ echo "# This has been generated by prepare.sh"
-	echo "no-stats";
-	echo "pf0 $(get_ifname 0)";
-	echo "pf1 $(get_ifname 1)";
-	echo "vf-pattern $(get_pattern ${devs[0]})";
-	echo "ipv6 $(get_ipv6)";
-	echo "a-pf0 ${devs[0]},class=rxq_cqe_comp_en=0,rx_vec_en=1,representor=pf[0]vf[0-$[$actualvfs-1]]";
-	echo "a-pf1 ${devs[1]},class=rxq_cqe_comp_en=0,rx_vec_en=1"; } >> "$CONFIG"
+	echo "no-stats"
+	echo "pf0 $(get_ifname 0)"
+	echo "pf1 $(get_ifname 1)"
+	echo "vf-pattern $(get_pattern ${devs[0]})"
+	echo "ipv6 $(get_ipv6)"
+	if [[ "$OPT_MULTIPORT" == "true" ]]; then
+		echo "a-pf0 ${devs[0]},class=rxq_cqe_comp_en=0,rx_vec_en=1,dv_flow_en=2,dv_esw_en=1,fdb_def_rule_en=1,representor=pf[0-1]vf[0-$[$actualvfs-1]]"
+		if [[ "$OPT_PF1_PROXY" == "true" ]]; then
```

Comment:
There is no indication on the log output that PF1 proxy was enabled. Would be nice to have it in the logs.

Reply:
Good point. some equivalent to the already present "enabling multiport eswitch mode for $pf". The only "problem" is that the script actually doe not do anything with the tap device it only sets it to enabled and chooses the MAC address.
So I guess just add `enabling pf1 proxy using tap device $mac`?

Reply:
Sounds good.